src/acthudweapon.cpp

```diff
@@ -3192,11 +3194,12 @@ void actHudShield(Entity* my)
 
 	bool defending = false;
 	bool sneaking = false;
-	if (!command && !swimming)
+    const bool shootmode = players[HUDSHIELD_PLAYERNUM]->shootmode;
+	if (!command && !swimming && shootmode)
 	{
 		if ( players[HUDSHIELD_PLAYERNUM] && players[HUDSHIELD_PLAYERNUM]->entity 
-			&& (*inputPressedForPlayer(HUDSHIELD_PLAYERNUM, impulses[IN_DEFEND]) 
-				|| (players[HUDSHIELD_PLAYERNUM]->shootmode && inputs.bControllerInputPressed(HUDSHIELD_PLAYERNUM, INJOY_GAME_DEFEND)))
+			&& input.binaryToggle("Block")
```

Comment:
Noting you can't hold block/sneak while in inventory anymore with 'space' - Can't remember if I ever use that in-game.

This is also the check for 'sneaking', so if we want separate "Block" / "Sneak" keys then another code path needs to handle sneaking